### ISSUE TITLE
[Package Request] autodl-irssi

### ISSUE THREAD

**Issue by USRGH** [SynoCommunity/spksrc#3283]
Name:autodl-irssi
Description: autodl-irssi is a plugin for irssi that monitors IRC announce channels and downloads torrent files based on user-defined filters.
Website: https://github.com/autodl-community/autodl-irssi
Software documentation: https://autodl-community.github.io/autodl-irssi
Build/Installation documentation: 
Source Code: 
License:

**Comment by ymartin59** [comment]
Probably it make sense to include it in `irssi` package...

**Comment by USRGH** [comment]
yes it will be perfect I think. With plugin to rTorrent /rutorrent of course.

**Comment by mreid-tt** [comment]
@ghost, I have prepared a new PR #6863 which includes the plugin as requested along with a package update. I am not familiar with this package so I'd appreciate it if you can help test it.
